### PR TITLE
Add 'grains_refresh_every' config on minions

### PR DESCRIPTION
**Pull request by grahamhayes**

### CODE REVIEW

File: app/views/dashboard/autoyast.xml.erb [hunk 1]
```diff
@@ -10,6 +10,7 @@
         <source><![CDATA[
         #!/bin/sh
         echo "master: <%= @controller_node %>" > /etc/salt/minion.d/master.conf
+        echo "grains_refresh_every: 10" >> /etc/salt/minion.d/master.conf
```
Comment (ereslibre) [inline app/views/dashboard/autoyast.xml.erb]: Why not adding it to another file like `/etc/salt/minion.d/grains_refresh.conf`?

I'm okay with adding this temporarily here, but shouldn't this configuration be managed by salt itself? We can trigger a restart of the salt-minion when this configuration changes, in theory we only need to set the master here.

Comment (grahamhayes) [inline app/views/dashboard/autoyast.xml.erb]: The problem with doing it in salt, it that when it gets updated, we need to restart the minion. Doing that in a run will cut off the rest of the highstate / orchestration step currently in progress.

Comment (ereslibre) [inline app/views/dashboard/autoyast.xml.erb]: Not sure if it supports a `SIGHUP`-like signal that only reloads the config...

Approving though.

**Comment by grahamhayes** [comment]
@ereslibre I updated the file name - good catch

File: app/views/dashboard/autoyast.xml.erb [hunk 2]
```diff
@@ -9,7 +9,8 @@
         <interpreter>shell</interpreter>
         <source><![CDATA[
         #!/bin/sh
-        echo "master: <%= @controller_node %>" > /etc/salt/minion.d/master.conf
+        echo "master: <%= @controller_node %>" > /etc/salt/minion.d/minion.conf
```
Comment (ereslibre) [inline app/views/dashboard/autoyast.xml.erb]: About this filename... it was fine as `master.conf` IMHO like: this is where the master is... (contents are: `master: location`)

Comment (kiall) [inline app/views/dashboard/autoyast.xml.erb]: minion.conf is used on the admin node... So this at least makes them consistent. Lets change them later, as this is blocking update testing etc

**Pull request merged**